pycqed/measurement/detector_functions.py

```diff
@@ -1317,14 +1317,14 @@ def prepare(self, sweep_points):
         # The averaging-count is used to specify how many times the AWG program
         # should run
         self.UHFQC.awgs_0_single(1)
-        self.UHFQC.qas_0_integration_length(int(self.integration_length*(1.8e9)))
+        self.UHFQC.qas_0_integration_length(
+            int(self.integration_length*(1.8e9)))
 
         self.UHFQC.qas_0_result_source(self.result_logging_mode_idx)
-        self.UHFQC.qudev_acquisition_initialize(channels=self.channels, 
-                                          samples=self.nr_shots*self.nr_sweep_points,
-                                          averages=1, #for single shot readout
-                                          loop_cnt=int(self.nr_shots),
-                                          mode='rl')
+        self.UHFQC.qudev_acquisition_initialize(
+            channels=self.channels, samples=self.nr_shots*self.nr_sweep_points,
+            averages=1, #for single shot readout
+            loop_cnt=int(self.nr_shots), mode='rl')
```

Comment:
I guess it is not the scope of this PR, but the init here is very similar to the integration_logging_detector. This class should maybe inherit from the logging_det to clean up the code.

Reply:
I made this class inherit from the UHFQC_integration_logging_det and cleaned it up a bit.